server/app.py

```diff
@@ -47,5 +49,16 @@ def quote(id):
     quote = Quote.query.get(id)
     return jsonify(quote.serialize)
 
+@app.route('/run_backtester')
+def run_backtester(algorithm, params, tickers):
```

Comment:
Does that work? I think you would need to allow POST with `methods=['POST']` and then get the POST parameters with the Flask `request` helper. 

Something like:
```python
from flask import request
#...
@app.route('/backtester/run', methods=['POST'])
    def run_backtester():
        data = request.get_json()
        #...
```